packages/web/components/Landing/LandingHeader.tsx

```diff
@@ -1,23 +1,114 @@
+/* eslint-disable no-nested-ternary */
 import {
   Box,
   BoxedNextImage,
   Button,
   Flex,
   HStack,
+  Icon,
+  IconButton,
   Link,
   Stack,
+  Text,
+  Tooltip,
+  usePrefersReducedMotion,
+  useToast,
   VStack,
 } from '@metafam/ds';
 import OctoBg from 'assets/baby_octo.png';
 import MetaGameLogo from 'assets/logo.png';
 import { MetaLink } from 'components/Link';
 import { SetStateAction } from 'jotai';
-import { Dispatch, ReactNode, useState } from 'react';
+import { get, set } from 'lib/store';
+import { Dispatch, ReactNode, useCallback, useEffect, useState } from 'react';
+import { FaToggleOff, FaToggleOn } from 'react-icons/fa';
+import { MdClose } from 'react-icons/md';
 
 import { AnimatedWaves, upDownAnimation } from './animations';
+import { LandingConnectButton } from './LandingConnectButton';
 
 export const LandingHeader: React.FC = () => {
   const [toggle, setToggle] = useState(false);
+  const prefersReducedMotion = usePrefersReducedMotion();
+  const savedMotionPreference = get('MotionPreference');
+  const [noMotion, setNoMotion] = useState(
+    savedMotionPreference === 'off'
+      ? true
+      : savedMotionPreference === 'on'
+      ? false
+      : prefersReducedMotion,
+  );
+  const reducedNoticeDismissed = get('ReducedMotionNotice') === 'dismissed';
+  const root = typeof window !== 'undefined' ? document.body : null;
+  const [effectsToggle, setEffectsToggle] = useState(noMotion);
+  const toggleIcon = effectsToggle ? FaToggleOff : FaToggleOn;
+  const [noticeOpen, setNoticeOpen] = useState(false);
+  const toast = useToast();
+
+  const handleCloseNotice = useCallback(() => {
+    setNoticeOpen(false);
+    set('ReducedMotionNotice', 'dismissed');
+  }, []);
+
+  /** TODO: Toggle works 100% except on first load when
+   * the switch renders as 'on' but is actually 'off' if no motion pref is saved */
+  const handleToggleEffects = useCallback(() => {
+    set('MotionPreference', !noMotion ? 'off' : 'on');
+    setNoMotion(!noMotion);
+    setEffectsToggle(!effectsToggle);
+    toast({
+      title: `Motion ${noMotion ? 'enabled' : 'disabled'}`,
+      description: `Toggle to turn effects & animations ${
+        noMotion ? 'off' : 'on'
+      }.`,
+      status: 'info',
+      duration: 5000,
+      isClosable: true,
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [effectsToggle, noMotion]);
+
+  /** Initially set the motion pref if it's not already set */
+  useEffect(() => {
+    if (typeof window !== 'undefined' && window.matchMedia !== undefined) {
+      if (savedMotionPreference === null) {
+        set('MotionPreference', prefersReducedMotion ? 'off' : 'on');
+      }
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
lets remove all occurences of `eslint-disable-next-line react-hooks/exhaustive-deps` and add the necessary dependencies into it.